spec/models/flipflop_spec.rb

```diff
@@ -5,8 +5,19 @@
     subject { described_class.assign_admin_set? }
     it { is_expected.to be true }
   end
+
   describe "enable_mediated_deposit?" do
     subject { described_class.enable_mediated_deposit? }
     it { is_expected.to be false }
   end
+
+  describe "proxy_deposit?" do
+    subject { described_class.proxy_deposit? }
+    it { is_expected.to be true }
+  end
+
+  describe "transfer?" do
```

Comment:
The spec you are describing appears to be `default value of transfer?` It may help legibility to explain that.